manifests/v1alpha2/katib-controller/crds.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: experiments.kubeflow.org
+spec:
+  group: kubeflow.org
+  version: v1alpha2
+  scope: Namespaced
+  names:
+    kind: Experiment
+    singular: experiment
+    plural: experiments
```

Comment:
Should we add condition in Experiment crd, like here: https://github.com/kubeflow/katib/blob/master/manifests/v1alpha1/studyjobcontroller/crd.yaml#L13 ?

Reply:
we can add it when experiment conditions done. now, we cannot know which one condition we should show